### PR TITLE
feat: add course to attendee

### LINKED ISSUE #533: Fix edit profile mode
When editing the profile, "username" input text changes to white (and should stay in black).

### PR DESCRIPTION
**Pull request by joaodiaslobo**
Add course selection to user on signup and edit profile, also fixes #533 since some changes were needed to add the new select course input to user profile. Depends on safira [#359](https://github.com/cesium/safira/pull/359).
 
 
image

image

### CODE REVIEW

**Comment by joaodiaslobo** [comment]
[feat: add course to attendee](https://trello.com/c/GEWIEKmt/51-feat-add-course-to-attendee)

**Review by MarioRodrigues10 — CHANGES_REQUESTED** [review]
Please add the degree that an attendee is currently in (Bachelors, Masters or PhD)

File: components/Input/index.tsx [hunk 1]
```diff
@@ -51,7 +43,9 @@ export default forwardRef<HTMLInputElement, Props>(function Input(
           autoComplete={autocomplete}
           value={value}
           required
-          className={`text-iregular ${textColor} ${backColor} block w-full appearance-none rounded-full border border-gray-300 px-3 py-2 pl-6 placeholder-gray-400 shadow-sm focus:outline-none sm:text-sm`}
+          className={`text-iregular ${
+            enabled == false ? "text-gray-500" : textColor
```
Comment (ruioliveira02) [inline components/Input/index.tsx:47]:
```suggestion
            enabled ? textColor : "text-gray-500"
```

File: pages/register/[uuid].js [hunk 2]
```diff
@@ -32,7 +32,7 @@ function Register() {
       password,
       password_confirmation,
       nickname,
-      uuid,
+      course,
```
Comment (ruioliveira02) [inline pages/register/[uuid].js:35]: I might be missing something, but shouldn't you need the UUID here as well?

Comment (joaodiaslobo) [reply pages/register/[uuid].js:35]: That's a mistake, thanks for the heads up! 🙏

**Review by ruilopesm — APPROVED** [review]

**Pull request merged**